fuzzers/fuzzbench_forkserver/src/main.rs

```diff
@@ -346,7 +349,8 @@ fn fuzz(
         cmplog_shmem.write_to_env("__AFL_CMPLOG_SHM_ID").unwrap();
         let cmpmap = unsafe { cmplog_shmem.as_object_mut::<AFLppCmpMap>() };
 
-        let cmplog_observer = StdCmpObserver::new("cmplog", cmpmap, true);
+        let cmplog_observer: StdCmpObserver<'_, _, _, CmpValuesMetadata> =
```

Comment:
The generics look a bit unfortunate, any way to get around them?

Reply:
How about having a `type StdCmpValuesObserver<'a, B, C> = StdCmpObserver<'a, B, C, CmValuesMetadata>` or something like that? Just an idea